response.h

```diff
@@ -0,0 +1,47 @@
+#ifndef RESPONSE_H
+#define RESPONSE_H
+
+#include <memory>
+#include <string>
+#include <boost/tokenizer.hpp>
+#include <map>
+
+// Represents an HTTP response.
+//
+// Usage:
+//   Response r;
+//   r.SetStatus(RESPONSE_200);
```

Comment:
Does our [enum](https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/49/files#diff-4ce81ec3dde55b3a00d9839d7bf10049R24) below need to be updated to match this enum value example? How strict is the API requirement on that? 

UPDATE: It doesn't look [any other team](https://github.com/search?q=org%3AUCLA-CS130+RESPONSE_200&type=Code) actually used `RESPONSE_200`. Even the example enum given has `OK = 200`. This is an inconsistency in the spec, and we should just stick to `OK` for brevity.